server/controllers/index.js

```diff
@@ -58,6 +63,19 @@ router.get('/api/start', auth, (req, res) => {
     res.sendStatus(200);
 });
 
+/* Check if player is logged in */
+router.get('/api/end', auth, (req, res) => {
+    const user = currentUsers.find((user) => user.username === req.session.user);
+    currentUsers = currentUsers.filter((otherUser) => otherUser !== user);
```

Comment:
If you're ending the game, shouldn't you be setting `currentUsers = []` rather than just removing the player that made the request?